CHANGELOG.md

```diff
@@ -82,8 +89,6 @@ aws lambda invoke --function-name $PREFIX-data-migration1 \
 
 ### Fixed
 
-- Fixed IAM permissions issue with `<prefix>-postgres-migration-async-operation` Lambda
```

Comment:
It is in [Release v10.1.0](https://github.com/nasa/cumulus/releases/tag/v10.1.0).  We should keep it.

Reply:
yes.  But fixes in patch v10.0.1 are not necessarily in the minor version.  This fix is in Release v10.1.0 and has been documented.

Reply:
I have been wondering why we release 10.1.1 and not 10.2.0

Reply:
I removed that fix note out of v10.1.0 section due to Mark's comment here https://github.com/nasa/cumulus/pull/2880#discussion_r819907812. So just want to make sure the update for this comment is not any conflict before proceeding. Thanks Jenny :)

Reply:
Then we should remove the note as well:
https://github.com/nasa/cumulus/pull/2884/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR104